src/components/Modal/README.md

```diff
@@ -5,102 +5,305 @@ Use the modal component to enter the user into a new _mode_.
 ```vue
 <template>
 	<div>
-		<demo />
-
+		<m-button
+			size="small"
+			@click="openModal"
+		>
+			Open modal
+		</m-button>
 		<m-modal-layer />
 	</div>
 </template>
 
 <script>
+import { MButton } from '@square/maker/components/Button';
 import { MModalLayer } from '@square/maker/components/Modal';
-import Demo from 'doc/Demo.vue';
+import DemoModal from 'doc/DemoModal.vue';
 
 export default {
+	name: 'DemoSetup',
+
 	components: {
-		Demo,
 		MModalLayer,
+		MButton,
 	},
 
 	mixins: [
 		MModalLayer.apiMixin,
 	],
+
+	methods: {
+		openModal() {
+			this.modalApi.open(() => <DemoModal />);
+		},
+	},
 };
 </script>
 ```
 
-_Demo.vue_
+_DemoModal.vue_
 
 ```vue
+<template>
+	<m-modal>
+		<img
```

Comment:
There's a couple concerns with leveraging this placeholder image.

1. We might want to use our image component here in the demo so there's a nice loading state as the image is downloading
2. This behavior of the width of the modal being skinny and then popping to be wider when the image loads isn't ideal. This will happen on production unless we can either recommend how to style the modal to support the output in a way that it doesn't happen.

A CSS only solution that would get us closer to the current desired output would look like this:
```
.cover-photo {
	width: 100%;
	max-height: 30vh;
	aspect-ratio: 18 / 9;
	object-fit: cover;
	object-position: center;
}
```

Aspect ratio isn't fully supported yet, so that's why it might be nice to build it into the Image component with fallbacks -- that doesn't need to be done in this PR, but try the code out -- it provides a better experience.